src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
```

Comment:
Add a getter for 'logo' property in Header class. Example: `get logo() { return 'Conduit logo'; }`